src/assets/alignmentD3Tests.js

```diff
@@ -124,37 +137,77 @@ var getXTickLocation = function(tick) {
  * @param {String} axisTick - innerHTML from text of a single tick. Ex: "37:00"
  * @returns {Number} float
  */
-var getTickValueMinutes = function(axisTick) {
+export function getTickValueMinutes(axisTick) {
   return parseInt(axisTick.split(':')[0], 10) +
     (parseInt(axisTick.split(':')[1], 10) / 60);
 };
 
 /**
- * Convert feature data-yvalue attribute from date object to to decimal minutes
+ * Parse one axis tick's label innerHTML
+ * @function
+ * @param {String} axisTick - innerHTML from text of single tick. Ex: "1990"
+ * @returns {Number} integer
+ */
+export function getTickValueInteger(axisTick) {
+  return parseInt(axisTick, 10);
+};
+
+/**
+ * Convert one axis tick's label innerHTML from month string to index
+ * @function
+ * @param {String} axisTick - innerHTML from text of a single tick. Ex: "March"
```

Comment:
This function assumes the time being formatted as %B (see https://github.com/d3/d3-3.x-api-reference/blob/master/Time-Formatting.md) which is "full month name". This is included in the heat map tests (see test #<span>11 on line 234 in file heat-map-tests.js). I think this would be more  reliable if you declare the `months` const only in heat-map-tests.js and pass it as a parameter to `isAxisAlignedWithDataPoints`. This is a case of DRY since the `months` const is declared in two places and should be declared in only one place.